src/Nethermind/Nethermind.Serialization.Rlp/RlpStream.cs

```diff
@@ -385,136 +385,48 @@ public void Encode(byte value)
             }
         }
 
-        public void Encode(bool value)
-        {
-            Encode(value ? (byte)1 : (byte)0);
-        }
+        public void Encode(bool value) => Encode(value ? (byte)1 : (byte)0);
 
-        public void Encode(int value)
-        {
-            Encode((long)value);
-        }
+        public void Encode(int value) => Encode((long)value);
```

Comment:
```suggestion
        public void Encode(int value) => Encode((ulong)value);
```

Reply:
Would need to be `(ulong)(long)value` due to negative number extension behaviour